### ISSUE TITLE
Avoid unnecessary map zooms when moving back to search page from other pages

### ISSUE THREAD

**Issue by lraivio** [City-of-Helsinki/varaamo#678]

**Comment by YoYuUm** [comment]
I understand this issue means that the map should keep the map position after the user comes back to the map from another page. I'm not sure if that's so simple because the map component is unmounted and mounted back, so I guess I should save the state of the map on map unmount and recover it on map mount in order to get that to work, unless you have some other idea

**Comment by lraivio** [comment]
Yeah something like that. This is a low priority issue however, so implement only if quick to do.

**Comment by lraivio** [comment]
No longer a problem after #727 was implemented.
